assessments/0010-etcd/etcd-analysis.md

```diff
@@ -0,0 +1,540 @@
+
+**In scope:**
+- Website: https://etcd.io
+- Documentation: https://etcd.io/docs
+- Website repo: https://github.com/etcd-io/website
+- Main project contributor info: https://github.com/etcd-io/etcd
+- Demo server: http://play.etcd.io
+
+**Out of scope:**
+- Other etcd repos: https://github.com/etcd-io/*
+
+
+## How this document is organized
+
+This document is divided into three sections that represent three major areas of concern:
+
+- **Project documentation:** concerns documentation for users of the etcd software, aimed at people who intend to use it
+- **Contributor documentation:** concerns documentation for new and existing contributors to the etcd OSS project
+- **Website:** concerns the mechanics of publishing the documentation, and includes branding, website structure, and maintainability
+
+Each section begins with summary ratings based on a rubric with appropriate [criteria][cncf-doc-criteria] for the section, then proceeds to:
+- **Comments**: observations about the existing documentation, with a focus on how it does or does not help etcd users achieve their goals.
+- **Recommendations**: suggested changes that would improve the effectiveness of the documentation. 
+
+An accompanying document, [etcd-implementation.md][implementation-doc], breaks the recommendations down into concrete actions that can be implemented by project contributors. Its focus is on drilling down to specific, achievable work that can be completed in constrained blocks of time. Ultimately, the implementation items should be tracked as a series of Github [issues][etcd-issues].
+
+
+## How to use this document
+
+Readers interested only in actionable improvements should skip this document and read [etcd-implementation.md][implementation-doc].
+
+Readers interested in the current state of the documentation and the reasoning behind the recommendations should read the section of this document pertaining to their area of concern:
+
+- [Project documentation][proj-doc]
+- [Contributor documentation][contributor-doc]
+- [Website and documentation infrrastructure][website]
+
+Examples of CNCF documentation that demonstrate the analysis criteria are linked from the [Criteria][cncf-doc-criteria] specification. 
+
+
+### Recommendations, requirements, and best practices
+
+Notwithstanding the fact that this analysis measures documentation against CNCF project maturity standards, in most cases there is more than one way to do things. Few recommendations here are meant to be prescriptive. Rather, the recommended implementations represent the reviewers' experience with how apply documentation best practices. In other words, borrowing terminology from the lexicon of [RFCs][rfc-keywords], the changes described here should be understood as "recommended" or "should" at the strongest, and "optional" or "may" in many cases. Any "must" or "required" actions are clearly denoted as such, and pertain to legal requirements such as copyright and licensing issues. 
```

Comment:
This analysis measures documentation against CNCF project maturity standards, and suggests possible improvements. In most cases...